src/guide/lnurl_withdraw.md

```diff
@@ -61,6 +61,23 @@ try {
 }
 ```
 </section>
+<div slot="title">Python</div>
+<section>
+
+```python
+# Endpoint can also be of the form:
+# lnurlw://domain.com/lnurl-withdraw?key=val
+lnurl_withdraw_url = "lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4exctthd96xserjv9mn7um9wdekjmmw843xxwpexdnxzen9vgunsvfexq6rvdecx93rgdmyxcuxverrvcursenpxvukzv3c8qunsdecx33nzwpnvg6ryc3hv93nzvecxgcxgwp3h33lxk"
+
+
+parsed_input = breez_sdk.parse_input(lnurl_withdraw_url)
+if isinstance(parsed_input, breez_sdk.InputType.LN_URL_WITHDRAW)
+    amount_sats = parsed_input.data.min_withdrawable
+    result = sdk_service.withdraw_lnurl(parsed_input.data, amount_sats, "comment")
```

Comment:
I haven't checked or python bindings yet, but there seems to be 2 different way we call API's: 
**breez_sdk** & **sdk_services**, what are the differences between them?

_(the latter seems to be a typo here when compared to majority)_
```suggestion
    result = sdk_services.withdraw_lnurl(parsed_input.data, amount_sats, "comment")
```

Reply:
The node functionality has to be called by the sdk_services by calling the `init_services`.  The `breez_sdk` is only the package in this case.